Description: Added keywords entry to desktop file to help end users search
Author: Franck Joncourt <franck.joncourt@gmail.com>
Applied-Upstream:1.4, commit:https://github.com/jp-bennett/fwknop-gui/commit/07b4fa293e48579f1c300ae43b415aeecb18a9c0

--- a/fwknop-gui.desktop
+++ b/fwknop-gui.desktop
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Name=Fwknop-gui
 GenericName=fwknop-gui
+Keywords=SPA;client
 Comment=send spa packets
 Exec=fwknop-gui
 Icon=fwknop-gui
